examples/qualcomm/qaihub_scripts/utils/README.md

```diff
@@ -0,0 +1,102 @@
+# CLI Tool for Compile / Deploy Pre-Built QNN Artifacts
+
+An easy-to-use tool for generating / executing .pte program from pre-built model libraries / context binaries from Qualcomm AI Engine Direct. Tool is verified with [host environement](../../../../docs/source/build-run-qualcomm-ai-engine-direct-backend.md#host-os).
```

Comment:
Is it generic for all models from ai hub?

Reply:
Yes, artifacts from AIHUB related to QNN are delivered with `.so` format. Only large generative AI models are shipped with context binaries.
Both of them could be transformed into `.pte` program with this tool.